### DISCUSSION TITLE
Why do you need the payment source id ?

### DISCUSSION THREAD

**Discussion by drewold123** [Vedza/NitroSniperGo#119] [Q&A] [answered]
func getPaymentSourceId() {
	var strRequestURI = []byte("https://discord.com/api/v8/users/@me/billing/payment-sources")
	req := fasthttp.AcquireRequest()
	req.Header.Set("authorization", settings.Maintoken)
	req.Header.SetMethodBytes([]byte("GET"))
	req.SetRequestURIBytes(strRequestURI)
	res := fasthttp.AcquireResponse()

whats the point of this code?

nevermind i think i got it

**Comment by Vedza** [comment, marked as answer]
To not have payment source required when redeeming nitro
